DatabaseMail - Display Name - Caractere "\" (#10203)
MicrosoftDocs/sql-docs: docs/relational-databases/database-mail/configure-database-mail.md

 **Display name**  
 Type the name to show on e-mail messages sent from this account. The display name is optional. This is the name displayed on messages sent from this account. For example, an account for [!INCLUDE [ssNoVersion](../../includes/ssnoversion-md.md)] Agent can display the name "SQL Server Agent Automated Mailer" on e-mail messages.  

>  If the display name contains backslash characters (`\`), you must escape them by using double backslashes (`\\`). For example, to display `SERVER\SQL`, enter `SERVER\\SQL` in the **Display Name** field. Single backslashes are interpreted as escape characters and will not appear in sent emails.

Reviewer: ```suggestion
 If the display name contains backslash characters (`\`), you must escape them by using double backslashes (`\\`). For example, to display `SERVER\SQL`, enter `SERVER\\SQL` in the **Display Name** field. Single backslashes are interpreted as escape characters and won't appear in sent emails.
```
